ldapadmin/src/main/filtered-resources/WEB-INF/views/privateUi.jsp

```diff
@@ -47,7 +47,7 @@ try {
     <meta charset="UTF-8">
     <title>LDAPadmin - <%= instanceName %></title>
     <link rel="stylesheet" href="lib/bootstrap/css/bootstrap.min.css" />
-
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/css/select2.css"/>
```

Comment:
We voted against CDNs on https://github.com/georchestra/georchestra/issues/939 
I'd prefer to have this resource locally instead (same thing below)

I would not be opposed to your PR fixing this other issue at the same time.